_test/events.test.js

```diff
@@ -0,0 +1,21 @@
+const eventEmitter = require('../events');
+
+
+describe('Event Logs', () => {
+  let consoleSpy;
+
+  beforeEach(() => {
+    // Attach to the console
+    consoleSpy = jest.spyOn(console, 'log').mockImplementation();
+  });
+
+  afterEach(() =>{
+    // put the console back
+    consoleSpy.mockRestore();
+  });
+
+  it('console logs pickup event', () => {
+    
+    expect(consoleSpy).toHaveBeenCalled();
+  })
+})
```

Comment:
Great direction with the test but it currently isn't passing, it looks like it is expecting at least one console.log but isn't picking up any console logs